teams/vsa/design/vsa-research-process.md

```diff
@@ -91,6 +95,8 @@ The maximum given to you according to our contract to synthesize research is “
 
 The following process is recommended by the VSA Design Lead:
 1. At the end of each day of research, schedule a “debrief” session where you invite all the observers to share the key takeaways from the sessions. Keep track of these key takeaways.
+    * On the right side of the Mural template, there is a Top Observation section, which is handy during the debrief sessions.  A benefit of this is team cohesion and getting folks interested in the research
+    * Use To Observations to discuss "what is top of mind about participant X" with your observers? No need to get a super thorough synthesis, just what sticks out based on what we heard. 
```

Comment:
this bullet is a little awkwardly worded?

Reply:
Ah yes - it was supposed to say **Top Observations**, not To observations. How about:

Use the Top Observation section to capture what your observers feel sticks out from each session. There is no need to get a super thorough synthesis at this stage.